src/aspire/image/__init__.py

```diff
@@ -179,7 +179,9 @@ def rotate(self):
 
     def save(self, mrcs_filepath, overwrite=False):
         with mrcfile.new(mrcs_filepath, overwrite=overwrite) as mrc:
-            mrc.set_data(self.data.astype('float32'))
+            # change back to the original input format (the image index first)
+            # for the consistency with reading
+            mrc.set_data(np.swapaxes(self.data.astype('float32'), 0, 2))
```

Comment:
Do we swap the other axes (the x and y axes) as well? This is something that we discussed earlier, but I don't remember what the conclusion was.

Reply:
I think it is not done yet. Issue #39 was opened for this problem. The change here only makes the input and output consistent.